### DISCUSSION TITLE
Add a sidebar (See attachment) to vanish

### DISCUSSION THREAD

**Discussion by APTminer1** [Zrips/CMI#7058] [Feature Requests]
We're going away from PremiumVanish to CMI Vanish, but we would miss a sidebar where they show things like playerinfo, names, TPS ping etc. We would greatly appreciate if something similar were to be implemented for Vanish. 
![Screenshot_1](https://user-images.githubusercontent.com/59396698/178561248-0cb059b1-349a-4afa-9a4c-aff65a3314d3.png)

**Comment by mrfloris** [comment]
just saw this, i replied a minute ago to this thread here with some config suggestions; yeah i'd love a very simple scoreboard - https://github.com/Zrips/CMI/discussions/7185 I don't need a full fledged complex plugin for loads of stuff.
